src/tests/graphqlish/eoblist.test.js

```diff
@@ -0,0 +1,112 @@
+const supertest = require('supertest');
+const {app} = require('../../app');
+// provider files
+const ExplanationOfBenefitResource = require('./fixtures/eob.json');
+const ExplanationOfBenefitResource2 = require('./fixtures/eob2.json');
+const ExplanationOfBenefitResource3 = require('./fixtures/eob3.json');
+// expected
+const expectedSingleExplanationOfBenefitResource = require('./fixtures/expected_eoblist.json');
+
+const async = require('async');
+
+const request = supertest(app);
+const { commonBeforeEach, commonAfterEach, getHeaders } = require('../common');
+
+describe('ExplanationOfBenefitGraphQLishTests', () => {
+  beforeEach(async () => {
+    await commonBeforeEach();
+  });
+
+  afterEach(async () => {
+    await commonAfterEach();
+  });
+
+  describe('ExplanationOfBenefit Search GraphQLish Test', () => {
+    test.skip('search all eobs with graphqlish like request', async () => {
+      await async.waterfall([
```

Comment:
disabled at the moment  not sure why this test times out, hopefully you can see something missing here

Reply:
usually that means the async/await is not used properly so the test exits before the work is done.  the other reason is of course that the query takes too long

Reply:
core team is working on a different approach here https://github.com/icanbwell/fhir-server/pull/273, this pr has trying to be unobtrusive to other functionality and because of that has run into challenges with auth, so this pr should be kept as a reference to the business logic being ported from typescript to js and the above approach is likely a better way to avoid duplication of effort and work off of that to encapsulate the business logic here under the correct auth structure.